Solutions/Network Session Essentials/Hunting Queries/DetectPortMisuseByAnomalyHunting.yaml

```diff
@@ -1,6 +1,9 @@
 id: 906c20c6-b62c-4af7-be91-d7300e3bded2
 name: Detect port misuse by anomaly (ASIM Network Session schema)
 description: |
+  'Detects anomalous port usage with ASIM normalization and is applied to sources supporting ASIM Network Session schema. Tune with 'NetworkSession_Monitor_Configuration' watchlist. The query uses summarized data for performance.'
```

Comment:
"This hunting query detect anomalous pattern in port usage with ASIM normalization. To tune the query to your environment configure it using the 'NetworkSession_Monitor_Configuration' watchlist."

Reply:
Resolved